warehouse/email.py

```diff
@@ -0,0 +1,37 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyramid_mailer import get_mailer
+from pyramid_mailer.message import Message
+
+from warehouse import celery
+
+
+@celery.task(bind=True, ignore_result=True, acks_late=True)
```

Comment:
I wonder if we really want ``acks_late`` here? The trade off is basically what behavior do we want on certain failures (particularly, a worker node dying). With ``acks_late=True`` we're almost certainly going to deliver the message at least once, but we might deliver it 2 or more times. With ``acks_late=False`` we're only going to deliver the message at most once, but we might deliver it zero times.

So the underlying question becomes then whether it's better to send extra emails in failure conditions or send no emails in failure conditions.